.github/workflows/e2e.yml

```diff
@@ -59,14 +59,6 @@ jobs:
             "state": "failure",
             "description": "E2E tests failed"
           }'
-      - name: Set argos upload branch on master branch
-        run: echo "ARGOS_BRANCH=master" >> $GITHUB_ENV
-        if: github.ref == 'refs/heads/main'
-      - name: Set argos upload branch on PR branches
-        run: echo "ARGOS_BRANCH=$(git name-rev --name-only $GITHUB_SHA | sed 's-remotes/origin/--')" >> $GITHUB_ENV
-        if: github.ref != 'refs/heads/main'
-      - name: Upload screenshots to Argos
-        env:
-          ARGOS_COMMIT: ${{ github.sha }}
-          ARGOS_TOKEN: ${{ secrets.ARGOS_TOKEN }}
-        run: yarn run argos upload e2e-screenshots
+      - name: Upload screenshots to argos-ci.com
+        if: always()
+        run: npx @argos-ci/cli upload e2e-screenshots
```

Comment:
So now the argos CLI detects correctly the branch even in the case of PR branches ?

Reply:
I guess ? https://github.com/argos-ci/argos-javascript/blob/bad0f5b5bf1dc1672b05cf08a4fec5d011ac4b0a/packages/core/src/ci-environment/git.ts#LL19C31-L19C62

Reply:
Yes, looks like!